CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 45.0.0
```

Comment:
People do usually add their changes to the Changelog in the first PR, but mark them as "Unreleased" instead of assigning them to a version, cf. [here](https://github.com/alphagov/govuk_content_models/pull/433/commits/3e7624ecf938f7fee3e7506de6354324e4ed5c88#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR3).